addons/better-terrain/editor/TileView.gd

```diff
@@ -2,6 +2,7 @@
 extends Control
 
 @onready var checkerboard := get_theme_icon("Checkerboard", "EditorIcons")
+@onready var dock = $"../../../../.."
```

Comment:
Indexing into the parent node path is a bit inflexible. I would rather replace this with appropriate signals, which are connected by the dock. Looks like you would need two signals, for `paste_occured` and `change_zoom_level` or something along those lines.

Reply:
good idea, i havent used signals a whole ton. let me know if the new changes look good (it works fine)